src/tree/azure/grouping/GroupingItem.ts

```diff
@@ -56,7 +56,7 @@ export class GroupingItem implements ResourceGroupsItem {
         } : undefined;
 
         if (this.context?.subscription) {
-            this.id = `/subscriptions/${this.context?.subscriptionContext.subscriptionId}/groupings/${this.label}`;
+            this.id = `/subscriptions/${this.context?.subscriptionContext.subscriptionId}/account/${this.context?.subscription.account?.id}/groupings/${this.label}`;
```

Comment:
Have we tested reveal to make sure it still works with this change?

Reply:
I have not tested it but I am pretty sure based on discussions with Nathan it would break things. I'll test it out and see.

Reply:
Ok, we'll need to sort that out before merging. I can help with that too

Reply:
Just tested it out by creating a storage account and clicking "click to view resource" in the activity log and nothing happened so seems to be broken. Before releasing this we definitely want to fix that.

Reply:
Bump